tests/ui/match_same_arms.rs

```diff
@@ -124,6 +124,20 @@ fn match_same_arms() {
         4 => 3,
         _ => 0,
     };
+
+    if true {
+        let foo = "";
+        return Ok(&foo[0..]);
+    } else if false {
+        let foo = "bar";
+        return Ok(&foo[0..]);
+    } else if true {
+        let foo = "";
+        return Ok(&foo[0..]);
+    } else {
+        let foo = "";
+        return Ok(&foo[0..]);
+    }
```

Comment:
Oh I think this should go into `if_same_then_else.rs`.

Also: It does not find the duplicates here - what did I do wrong here?

Reply:
syntax error